src/hmem_cuda.c

```diff
@@ -315,7 +315,7 @@ int cuda_copy_from_dev(uint64_t device, void *dst, const void *src, size_t size)
 int cuda_dev_register(const void *addr, size_t size, uint64_t *handle)
 {
 	if (cuda_is_gdrcopy_enabled())
-		return cuda_gdrcopy_dev_register(addr, size, handle);
+		return cuda_gdrcopy_dev_register((void *)addr, size, handle);
```

Comment:
is there any reason to not `const` the input of `cuda_gdrcopy_dev_register`? 
seems that it's a read-only parameter
https://github.com/ofiwg/libfabric/blob/95874ad4b5300ec6123aee178bf365211555c5da/src/hmem_cuda_gdrcopy.c#L345-L347C11

Reply:
No, no reason, we can do it in that way

Reply:
Updated.